NetFixClient.vcxproj

```diff
@@ -51,7 +51,8 @@
       <ProgramDataBaseFileName>.\ReleaseMinSize\</ProgramDataBaseFileName>
     </ClCompile>
     <PostBuildEvent>
-      <Command>Copy  ReleaseMinSize\NetFixV3.dll  C:\Sierra\Outpost2\NetFix\</Command>
+      <Command>
+      </Command>
```

Comment:
I think I might have preferred commenting this out with `REM` (remark). That way there's still a template of how to do the copy when testing.

Though perhaps there is a better way to do the testing.

Reply:
I deleted this because it is hardcoded and uses an absolute directory. I didn't think it was worth keeping around. 

I'll add a better post build event, but won't be able to test it until we get the code compiling.

-Brett

Reply:
Do you have a way of doing this without a hardcoded directory?

I'm thinking maybe an environment variable setting and a rule to conditionally copy files if the environment variable is present. Not sure how else to do it, since each system may be setup differently.